Cargo.toml

```diff
@@ -41,7 +41,7 @@ signal-hook = "0.3"
 slab = "0.4"
 socket2 = "0.4"
 thiserror = "1.0"
-time = { version = "=0.3.18", features = ["formatting", "local-offset", "macros"] }
```

Comment:
thanks for the udpate.
lets pin to the specific version `version "=0.3.36"`

Reply:
What's the advantage of pinning to a specific version? Most of the dependencies don't do it, so I guess the exceptions are there for a reason, but I don't recognize that reason for the `time` crate.